hudi-common/src/main/java/org/apache/hudi/common/table/log/block/HoodieLogBlock.java

```diff
@@ -271,9 +271,25 @@ public static byte[] getLogMetadataBytes(Map<HeaderMetadataType, String> metadat
   }
 
   /**
-   * Convert bytes to LogMetadata, follow the same order as {@link HoodieLogBlock#getLogMetadataBytes}.
+   * Convert footer metadata to bytes 1. Write size of metadata 2. Write enum ordinal 3. Write actual bytes
    */
-  public static Map<HeaderMetadataType, String> getLogMetadata(SeekableDataInputStream dis) throws IOException {
+  public static byte[] getFooterMetadataBytes(Map<FooterMetadataType, String> metadata) throws IOException {
```

Comment:
The `FooterMetadataType` is empty, is this just a code usage fix?

Reply:
> The `FooterMetadataType` is empty, is this just a code usage fix?

Yes, currently it is just a code usage fix.